client/scripts/sw-docker-entrypoint

```diff
@@ -44,6 +44,7 @@ prepare(){
         # only remote
         echo '-->[Preparing] pulling model ...'
         swcli instance login --token "${SW_TOKEN}" --alias server ${SW_INSTANCE_URI}
+        swcli project create ${SW_PROJECT} && swcli project select ${SW_PROJECT}
```

Comment:
create `${SW_PROJECT}` project in standalone ?

Reply:
This point is mainly considered when user submit a job to server(such as run in a project named 'demo-a'),but we final submit it to a container, then run it on the local project named 'self' in standalone by default but not 'demo-a'. If user use it to record something, it will be wrong

Reply:
removed